src/components/06_wrappers/Default/Default.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { func, node, arrayOf, shape, string } from 'prop-types';
+import PropTypes from 'prop-types';
```

Comment:
Seems like an unnecessary changes, but it also seems most people import proptypes like that.

Reply:
yeah I was trying to figure stuff out with defaultProps to start with and it was really annoying to work with in it's current state